content/applications/websites/ecommerce/products.rst

```diff
@@ -4,275 +4,402 @@
+Configuration --> Settings`, scroll down to the :guilabel:`Shop - Products` section and the
+:guilabel:`Inventory Defaults` sub-section.
 
 .. important::
-   To display the stock level on the product page, the :guilabel:`Product Type` on the **product's
-   form** must be set to :guilabel:`Storable` (only available when the **Inventory** app is
-   installed).
+   - The **Inventory** app must be installed to see the inventory management options.
+   - To display the stock level on your product page, the :guilabel:`Product Type` must be set to
+     :guilabel:`Storable` in the :ref:`product form <ecommerce-products-product-form>`.
+
+.. _ecommerce-products-inventory:
 
 Inventory
 ---------
 
-In the :guilabel:`Inventory Defaults` sub-section, you can select the eCommerce selling strategy of
-products:
+In the :guilabel:`Inventory Defaults` sub-section, fill in those fields:
 
-- :guilabel:`Warehouse`: if you have multiple warehouses, you can define the warehouse associated to
-  your website. If you have multiple websites, you can select a different one per website;
-- :guilabel:`Out-of-Stock (Continue Selling)`: enabling it allows customers to continue placing
-  orders even when the product is **out-of-stock**.
-  Leave it  unchecked to **prevent orders**;
-- :guilabel:`Show Available Qty`: enabling it displays the available quantity left under a specified
-  threshold on the product page. The available quantity is calculated based on the 'On hand'
+- :doc:`Warehouse <../../inventory_and_mrp/inventory/warehouses_storage/inventory_management/warehouses>`;
+- :guilabel:`Out-of-Stock`: enable :guilabel:`Continue Selling` to allow customers to place orders
+  even when the product is **out of stock**. Leave the box unchecked to **prevent orders**;
+- :guilabel:`Show Available Qty`: displays the available quantity left under a specified threshold
+  on the product page. The available quantity is calculated based on the :guilabel:`On hand`
   quantity minus the quantity already reserved for outgoing transfers.
 
-.. seealso::
-   :ref:`Allow only selected customers to buy <cart/prevent-sale>`
-
-Selling as kit
---------------
-
-If you are selling non-prepackaged kits (i.e., the kits are made of individual products), we
-recommend you read the related documentation to keep track of your stock.
-
-.. seealso::
-   :doc:`../../inventory_and_mrp/manufacturing/advanced_configuration/kit_shipping`
+.. _ecommerce-products-product-comparison:
 
 Product comparison
 ==================
 
-You can enable a **product comparison tool** for your eCommerce by going to
-:menuselection:`Website --> Configuration --> Settings --> Shop - Products`, and ticking
-:guilabel:`Product Comparison Tool`. This tool allows to save products' **specifications** and
-compare them against each other on a single page.
+To allow website visitors to compare products based on their attributes, go to
```

Comment:
I mentioned this in the general comment but this section is a good example of why it's best to avoid using the 2nd pronoun too much: In this case, you address the reader (i.e., in most cases, the person who configures the eCommerce app) *and* their customers (i.e. the people who will actually use the Compare tool) using *you*, which I think is a little confusing. I would actually refer customers onlu as *customers* when possible (see below for suggestions.